x-pack/plugins/cloud_security_posture/public/pages/findings/findings_flyout/resource_tab.tsx

```diff
@@ -6,102 +6,97 @@
  */
 
 import {
-  EuiAccordion,
-  EuiCode,
-  EuiCodeBlock,
-  EuiDescriptionList,
-  EuiPanel,
+  type EuiInMemoryTableProps,
+  EuiMarkdownFormat,
+  EuiInMemoryTable,
+  EuiToolTip,
   EuiSpacer,
   EuiText,
-  useEuiTheme,
+  EuiCode,
+  EuiCopy,
 } from '@elastic/eui';
-import React, { useMemo } from 'react';
+import React from 'react';
 import { getFlattenedObject } from '@kbn/std';
 import { i18n } from '@kbn/i18n';
 import { CspFinding } from '../../../../common/schemas/csp_finding';
 
-const getDescriptionDisplay = (value: unknown) => {
-  if (value === undefined) return 'undefined';
-  if (typeof value === 'boolean' || value === null) {
-    return <EuiCode>{JSON.stringify(value)}</EuiCode>;
-  }
+interface ResourceItem {
+  key: string; // flattened dot notation object path for CspFinding['resource'];
+  value: unknown;
+}
 
-  if (typeof value === 'object') {
-    return (
-      <EuiCodeBlock isCopyable={true} overflowHeight={300}>
-        {JSON.stringify(value, null, 2)}
-      </EuiCodeBlock>
-    );
+function renderField(record: ResourceItem) {
+  switch (record.key) {
+    case 'id':
+    case 'name':
+      return <EuiCode>{JSON.stringify(record.value, null, 2)}</EuiCode>;
+
+    default:
+      return <EuiMarkdownFormat>{record.value as string}</EuiMarkdownFormat>;
   }
+}
+
+const columns: EuiInMemoryTableProps<ResourceItem>['columns'] = [
+  {
+    field: 'key',
+    name: i18n.translate('xpack.csp.flyout.resource.fieldLabel', { defaultMessage: 'Field' }),
```

Comment:
i think those ids are not very specific, the previous format was `'xpack.csp.findings.findingsFlyout.resourceTab.` which i find better